recipes/scine-sparrow-python/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "scine-sparrow-python" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/qcscine/sparrow/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 70636871694c9363ae3fb2df5050bddb22667b71d875d5a7e9afd872f6a2b65d
+    fn: {{ name }}.tar.gz
+    patches:
+      - skip_library_build.patch
+  - url: https://github.com/qcscine/development-utils/archive/refs/tags/4.0.0.tar.gz
```

Comment:
would it make sense to distribute this as a seperate package if it is used across multiple packages?

Reply:
Those are some kind of common CMake files rather than an actual installable/distributable package, which are actually used as git submodule upstream but easier to include with multiple sources.

Reply:
This is fine.